var/spack/repos/builtin/packages/py-nipype/package.py

```diff
@@ -45,3 +45,5 @@ class PyNipype(PythonPackage):
     depends_on("py-etelemetry@0.2:", when="@1.5:", type=("build", "run"))
     depends_on("py-etelemetry", type=("build", "run"))
     depends_on("py-looseversion", when="@1.8.1:", type=("build", "run"))
+
+    depends_on("py-sphinx", when="@1.8.1:", type=("test"))
```

Comment:
We aren't actually running any unit tests, so I think you can remove this.

Reply:
Without this dependency `--test=root` does not work.
Btw. did anything change regarding the tests? When I added the last release for this package, meaning 1.8.5,  `--test` worked without this dependency. Now it doesn't anymore.

Reply:
Could be that the test files are now installed and we try to import them. I would add them to `skip_modules` instead of adding a dep.

Reply:
Ok, I have removed the test dependency and added the `skip_modules` instead.